src/test/scala/no/ndla/imageapi/service/search/SearchServiceTest.scala

```diff
@@ -70,32 +58,31 @@ class SearchServiceTest extends UnitSuite with TestEnvironment {
   }
 
   override def afterAll() = {
-    esNode.close()
-    Path(esDataDir).deleteRecursively()
+    indexService.deleteIndex(Some(ImageApiProperties.SearchIndex))
   }
 
-  test("That getStartAtAndNumResults returns default values for None-input") {
+  test("That getStartAtAndNumResults returns default values for None-input", IntegrationTest) {
```

Comment:
Tja...vet ikke helt om jeg er så fan av navnet `ESIntegrationTest`. ES henviser til elasticsearch, som da vil være misvisende om vi har integrasjonstester som ikke er spesifikke for elasticsearch. (Selv om i denne testen er det jo en integrasjonstest som bruke elasticsearch)